apiserver/metricsender/metricsender.go

```diff
@@ -23,8 +23,9 @@ type MetricSender interface {
 }
 
 var (
-	defaultMaxBatchesPerSend              = 10
-	defaultSender            MetricSender = &HttpSender{}
+	defaultMaxSendsPerInvocation              = 5
```

Comment:
I don't think we can choose an arbitrary limit here. It'd be better to stop sending when progress stalls: if no batches are being acknowledged and we'd send the same ones over and over.